src/store/initial_state.ts

```diff
@@ -220,20 +220,6 @@ const state: GlobalState = {
                 error: null,
             },
         },
-        preferences: {
-            getMyPreferences: {
-                status: 'not_started',
-                error: null,
-            },
-            savePreferences: {
-                status: 'not_started',
-                error: null,
-            },
-            deletePreferences: {
-                status: 'not_started',
-                error: null,
-            },
-        },
```

Comment:
The corresponding [state.requests](https://github.com/mattermost/mattermost-mobile/blob/master/app/initial_state.js#L127-L140) on mobile should be removed too one the mattermost-redux hash is updated.

Reply:
Follow up PR for mobile - https://github.com/mattermost/mattermost-mobile/pull/3836